WebClient/doc/debugging.md

```diff
@@ -0,0 +1,78 @@
+# Debugging hints
+
+## Trigger functionality from developer console
+
+In Chrome or Explorer or any other browser one can execute javascript code from Developer Console and trigger application functionality.
+
+An important detail is that classes can't be accessible directly from the global scope because our code is wrapped in modules. To access a class or any other object or function exported by some module you can use `require(modulePath)` function. This function is available in the global scope and it allows loading the specified module using its path relative to project folder, e.g `require('./src/components/graph/graphModel.js')`. **Note** that `require()` function is available in global scope only in debug build and not in production.
```

Comment:
typo? "accessible" should be "accessed"?

Reply:
or maybe: "can't be accessible directly" should be "are not directly accessible"?